mailbox/api/src/main/java/org/apache/james/mailbox/events/GroupsAlreadyRegistered.java

```diff
@@ -19,15 +19,6 @@
 
 package org.apache.james.mailbox.events;
 
-public class GroupAlreadyRegistered extends RuntimeException {
-    private final Group group;
+public class GroupsAlreadyRegistered extends RuntimeException {
```

Comment:
please suffix the type with `Exception` (yes, I know, it's painful but I think it's a coding standard we always respect)